src/js/files/components/UploadItem.tsx

```diff
@@ -15,7 +16,11 @@ const StyledUploadItem = styled.div`
     }
 `;
 
-const UploadItemTitle = styled.div`
+/**
+ * Styled component for the title section of the upload item.
```

Comment:
Please take a crack at updating these as mentioned in the main comment. You can use the new `UploadItemName` as an example. Happy to provide guidance

Reply:
The updated docstrings should be more aligned now, but let me know what you think

Reply:
Looks good, looks like your running into a `prettier` CI problem though. Run prettier on the file loaded and it should be good to go